collect_app/src/main/java/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -106,6 +107,9 @@
     @Inject
     SettingsProvider settingsProvider;
 
+    @Inject
+    MediaUtils mediaUtils;
```

Comment:
Again, I think we could avoid Dagger here and just use the constructor as before.

Reply:
We agree not to change it because that would require passing that object through the entire hierarchy of widgets (something we need to look into since it's not the first problem like that).